#! /bin/sh /usr/share/dpatch/dpatch-run
## 10-fix-jar-path.dpatch by  <fourmond@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix the path of the library jars to match those of the debian
## DP: packages 

@DPATCH@
Index: freecol-0.10.0+dfsg/build.xml
===================================================================
--- freecol-0.10.0+dfsg.orig/build.xml	2011-06-25 17:12:29.000000000 +0200
+++ freecol-0.10.0+dfsg/build.xml	2011-06-25 17:17:57.346240904 +0200
@@ -105,9 +105,10 @@
     <!-- Compiles the java source files. -->
     <target name="compile" depends="init">
         <mkdir dir="${freecol.build.dir}"/>
+	<!-- Classpath is provided by debian/rules, so commenting this -->
+	<!--             classpathref="freecol.build.classpath" -->
         <javac srcdir="${freecol.src.dir}"
             destdir="${freecol.build.dir}"
-            classpathref="freecol.build.classpath"
             debug="on"
             optimize="on"
             deprecation="off"
@@ -158,7 +159,7 @@
 
     <!-- Compiles the source files and creates a JAR-file. -->
     <target name="package" depends="init,build,manifest" description="Compiles the source files and creates a JAR-file.">
-        <jar jarfile="${freecol.jar.file}"
+        <jar jarfile="freecol.jar"
             basedir="${freecol.build.dir}"
             manifest="${basedir}/src/MANIFEST.MF"
             includes="net/**, org/**"
